### DISCUSSION TITLE
ETA for #59719 (closed recently)

### DISCUSSION THREAD

**Discussion by yaroslavlsd** [dotnet/roslyn#76224] [Q&A]
`[Project template] Blank line is not preserved after namespace when file has usings `

Still reproducible in VS 17.12.2.
I cant find commit in roslyn that fixed bug around Oct 25.

@CyrusNajmabadi Issue is closed, locked and limited conversation to collaborators

**Comment by CyrusNajmabadi** [comment]
Dupe of https://github.com/dotnet/roslyn/issues/59719

Reopening that one.

**Reply by yaroslavlsd** [reply]
Since you recently closed it (`Closing as fixed`), I understood that it was fixed recently and will be released soon in next versions, is it true?

**Reply by CyrusNajmabadi** [reply]
No.  It looks like there are potentially two different issues.  One with the templates being created outside of VS and one where it happens in VS.  It may be the case that the latter is still buggy.  I reactivated to check and see if the fix addresses both cases, or just one
